Library/Homebrew/load_path.rb

```diff
@@ -7,3 +7,5 @@
 $LOAD_PATH.push(HOMEBREW_LIBRARY_PATH.to_s) unless $LOAD_PATH.include?(HOMEBREW_LIBRARY_PATH.to_s)
 
 require "vendor/bundle/bundler/setup"
+
+$LOAD_PATH.uniq!
```

Comment:
In that case how about something like:

```suggestion
require "vendor/bundle/bundler/setup"

$LOAD_PATH.push HOMEBREW_LIBRARY_PATH.to_s
$LOAD_PATH.select! { |d| Pathname(d).directory? }
$LOAD_PATH.uniq!
```